src/test/java/picard/util/LiftoverVcfTest.java

```diff
@@ -1201,19 +1201,21 @@ public void testUnreadableDictionary() throws IOException {
         final Path rejectOutput = Files.createTempFile("tmpreject", ".vcf");
         rejectOutput.toFile().deleteOnExit();
         final Path input = TEST_DATA_PATH.toPath().resolve("testLiftoverBiallelicIndels.vcf");
+        final Path referenceCopy = Files.createTempFile("ref", ".fasta");
+        referenceCopy.toFile().deleteOnExit();
+        Files.copy(REFERENCE_FILE.toPath(), referenceCopy, StandardCopyOption.REPLACE_EXISTING);
```

Comment:
overwriting a temporary file?

Reply:
yeah, once _Files.createTempFile_ is called the file exists and _Files.copy_ will need to overwrite.  I can't really find a good way to get a temp path for the reference copy without creating the file; I could create a helper temp file and then _resolveSibling_ on the helper to get the reference copy, or use _Paths.get_ on an absolute path into the java tmp directory, but neither of those options seems better than just allowing the copy to overwrite.

Reply:
I think what you want to do is use `Files.createTempDirectory()` to create the temp directory and then `Path.resolve()` on it to create your new temp file path.

Reply:
I agree, that's better, thanks.